static/js/vue-cdr-access/src/mixins/routeUtils.js

```diff
@@ -0,0 +1,82 @@
+import isEmpty from 'lodash.isempty';
+
+export default {
+    methods: {
+        /**
+         * Put URL parameters into an object
+         * Set default params, if none are present that can be updated
+         * @param params_to_update
+         * @returns {({} & Dictionary<string | (string | null)[]>) | {start: number, page: number, sort: string, rows: number}}
+         */
+        urlParams(params_to_update = {}) {
```

Comment:
I think this whole method might be able to be refactored to the following algorithm:
```
if $route.query is empty
  then route_params = default values
  else route_params = $route.query

page_params = route_params
if params_to_update is NOT empty
  then page_params = assign(page_params, params_to_update)

return page_params
```